config/locales/room/en.yml

```diff
@@ -1,7 +1,7 @@
 en:
   rooms:
-    no_furniture_notice: 'No gizmos yet. Deck out this section by adding some gizmos!'
-    place_furniture_heading: 'Add a new gizmo'
+    no_furniture_notice: "No gizmos yet. Deck out this section by adding some gizmos!"
+    place_furniture_heading: "Add a new gizmo"
```

Comment:
This looks like it is not related to this change. I don't know that we have a strong policy re: Not including stuff like this in patches or not.

On the one hand, tidying as you go when you notice something that wants tidying is a lovely practice.

On the other hand, `git history` on this line will be a little bit less useful; which may or may not matter.